app/src/main/java/com/github/libretube/ui/fragments/SubscriptionsFragment.kt

```diff
@@ -258,15 +246,18 @@ class SubscriptionsFragment : DynamicLayoutManagerFragment() {
     }
 
     private fun List<StreamItem>.filterByStatusAndWatchPosition(): List<StreamItem> {
+
         val streamItems = this.filter {
             val isLive = (it.duration ?: -1L) < 0L
```

Comment:
We know have a `StreamItem#isLive` property, so this line is redundant and can be replaced with just using `it.isLive`.